Flash Chat iOS13/AppDelegate.swift

```diff
@@ -13,6 +13,8 @@ import Firebase
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
+	var userEmails: [String] = []
```

Comment:
❓ 
Very minor; just curious why Xcode isn't auto-adjusting the indentation here to match, for example, line 18 below.

```suggestion
    var userEmails: [String] = []
```

Reply:
Meant to put the array comment here instead of below...